### DISCUSSION TITLE
Interested to Contribute

### DISCUSSION THREAD

**Discussion by Mukku27** [labmlai/annotated_deep_learning_paper_implementations#269] [Show and tell]
Hi @vpj ,

I'm Mukesh Vemulapalli, an undergraduate student majoring in Electronics and Communication Engineering at Rajiv Gandhi University of Knowledge Technologies.

I'm interested in contributing to paper implementations related to GANs (Generative Adversarial Networks). If you have any feature requests or bugs that need attention, feel free to assign those issues to me. I would be happy to spend time reading the respective papers and work on resolving the issues.

I have worked on some interesting projects inspired by research papers, which you can find here:

- [CycleGAN - Unpaired Image Translation](https://github.com/Mukku27/CycleGAN-Unpaired-Image-Translation.git): Based on the research paper "Unpaired Image-to-Image Translation using Cycle-Consistent Adversarial Networks" ([arXiv link](https://arxiv.org/abs/1703.10593)).

- [Conditional GAN for Street-to-Satellite Image Translation](https://github.com/Mukku27/Conditional_GAN_for_Street-to-Satellite_Image_Translation.git): Inspired by the research paper "Image-to-Image Translation with Conditional Adversarial Networks" ([arXiv link](https://arxiv.org/abs/1611.07004)).

- [Unsupervised Representation Learning with DCGAN](https://github.com/Mukku27/Unsupervised-Representation-Learning-DCGAN.git): A basic implementation of a DCGAN for unsupervised representation learning. You can experiment with different hyperparameters (learning rate, batch size, network architectures) to improve the quality of the generated images. Check out the research paper [here](https://arxiv.org/pdf/1511.06434.pdf).

Looking forward to contributing to the community and learning from this experience.

Thank you
